distribution/docs/src/main/resources/content/_running/catalog-commands.adoc

```diff
@@ -0,0 +1,146 @@
+:title: Catalog Commands
+:type: subMaintaining
+:status: published
+:parent: Available Console Commands
+:summary: Catalog commands available.
+:order: 00
+
+== {title}
+
+[WARNING]
+====
+Most commands can bypass the Catalog framework and interact directly with the Catalog provider if given the `--provider` option, if available.
+No pre/post plugins are executed and no message validation is performed if the `--provider` option is used.
+====
+
+.[[catalog_command_descriptions]]Catalog Command Descriptions
+[cols="1m,9a" options="header"]
+|===
+
+|Command
+|Description
+
+|catalog:describe
+|Provides a basic description of the Catalog implementation.
+
+|catlog:dump
+|Exports metacards from the local Catalog. Does not remove them. See <<_date_filtering_options,date filtering options>> below.
+
+|catalog:envlist
+|[IMPORTANT]
+====
+Deprecated as of ddf-catalog 2.5.0. Please use `platform:envlist`.
+====
+
+Provides a list of environment variables.
+
+|catalog:ingest
+|Ingests data files into the Catalog. XML is the default transformer used. See <<_ingest_command,Ingest Command>> for detailed instructions on ingesting data and <<_input_transformers,Input Transformers>> for a detail
```

Comment:
Should this link be `_ingest_command_command_console` instead?

Reply:
Should this link be `_available_input_transformers` instead?

Reply:
For the first one, I missed a file edit for `_ingest_command` in this commit. It's fixed now.

For the second, Yes, and fixed.